molecule/default/prepare.yml

```diff
@@ -51,8 +58,13 @@
         regexp: "TEST_FQDN"
         replace: "{{ certificate_domain }}"
 
+    - name: Ensure that clone_github_token is correctly set
+      ansible.builtin.assert:
+        that:
+          - clone_github_token is defined
+          - clone_github_token.startswith('ghp_')
+
     - name: Git checkout
-      ansible.builtin.git:
-        repo: "https://{{ lookup('env', 'CLONE_GITHUB_TOKEN') }}@github.com/Alfresco/alfresco-deployment-test-automation-scripts.git"
-        dest: "{{ dtas_dir }}"
-        version: "{{ lookup('env', 'DTAS_VERSION') }}"
+      ansible.builtin.command:
+        cmd: "git clone --depth 1 --branch {{ dtas_version }} https://{{ clone_github_token }}@github.com/Alfresco/alfresco-deployment-test-automation-scripts.git {{ dtas_dir }}"
+        creates: "{{ dtas_dir }}"
```

Comment:
this needs to be investigate further, no surprise if it's still related to the /tmp mounted as noexec (just guessing)